azure-pipelines.yml

```diff
@@ -399,18 +368,12 @@ jobs:
   # Build CW310 variant of the Earl Grey toplevel design using Vivado
   dependsOn:
     - lint
-    # The bootrom is built into the FPGA image at synthesis time.
-    - cw310_sw_build
   condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'), eq(dependencies.lint.outputs['DetermineBuildType.onlyDvChanges'], '0'), eq(dependencies.lint.outputs['DetermineBuildType.onlyCdcChanges'], '0'))
   pool: ci-public
   timeoutInMinutes: 180
   steps:
   - template: ci/checkout-template.yml
   - template: ci/install-package-dependencies.yml
-  - template: ci/download-artifacts-template.yml
-    parameters:
-      downloadPartialBuildBinFrom:
-        - cw310_sw_build
```

Comment:
The cached bitstream flow is now broken. We'll need to fix prepare-cached-bitstream.sh to build the test_rom and splice it into the cached bitstream.

Reply:
Ah, so the cached bitstream flow broken because artifacts from `cw310_sw_build` are no longer available? In that case, could `prepare-cached-bitstream.sh` just be replaced with `ci/bazelisk.sh build --define bitstream=gcp_splice //hw/bitstream:test_rom`?

Reply:
I've not tried that rule, so I couldn't say, but that might do most of it. The original outputs of this job were a bitstream with a newly-built test_rom spliced in. The bitstream could be generated in this job or it could come from the cache.

However, the step that uses the cached bitstream ran the splice_bitstream.sh shell script, and that assumed the test_rom artifacts were already present. In addition, It looks like this job now _also_ has an output with the prod rom spliced in. The cached bitstream path will need to be synced up with this new job's flow.

Reply:
Thanks for catching this! I'm looking into this now, and it looks like it should be possible to find the right Bazel target(s) to match this up with what was there before. I think running something like `ci/bazelisk.sh build gcp_spliced_test_rom gcp_spliced_rom gcp_spliced_rom_otp_dev` and then doing some file-moving will probably work.

Reply:
Okay I've modified the cached bitstream flow forced it to run through CI. It looks like it's passing everything. Take a look at `prepare-cached-bitstream.sh` for the modifications.

Reply:
Thanks @milesdai !